src/display/editor/tools.js

```diff
@@ -377,6 +377,21 @@ class AnnotationEditorUIManager {
     }
   }
 
+  /**
+   * Update the toolbar if it's required to reflect the tool currently used.
+   * @param {number} mode
+   * @returns {undefined}
+   */
+  updateToolbar(mode) {
+    if (mode === this.#mode) {
+      return;
+    }
+    this.#eventBus?.dispatch("switchannotationeditormode", {
```

Comment:
I'd probably argue that the `#eventBus` being undefined here is a bug in code that's initializing the `AnnotationEditor`; so you might be able to simply assume that it's available?